app/controllers/clinical_resource_hubs_controller.rb

```diff
@@ -1,9 +1,60 @@
 class ClinicalResourceHubsController < ApplicationController
-  before_action :set_crh, only: [:show]
+  include PracticesHelper
+  include PracticeUtils
+  include StatesHelper
+  before_action :set_crh, only: [:show, :created_crh_practices]
   def show
+    visn_id = @visn.id
+    visn_number = @visn.number
+    @primary_visn_liaison = VisnLiaison.find_by(visn: @visn, primary: true)
```

Comment:
Do we need this? I don't think we're using it anywhere.

Reply:
Good catch... ur right we don't need those lines.